src/applications/static-pages/facilities/FacilityPatientSatisfactionScoresWidget.jsx

```diff
@@ -0,0 +1,129 @@
+              <p className="vads-u-margin--0">Primary care</p>
+              <p
+                id="facility-patient-satisfaction-scores-primary-urgent-score"
+                className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
+              >
+                {this.state.facility.attributes.feedback.health
+                  .primaryCareUrgent
+                  ? `${this.state.facility.attributes.feedback.health
+                      .primaryCareUrgent * 100}%`
+                  : 'N/A'}
+              </p>
+            </div>
+            <div className="usa-width-one-half vads-u-background-color--gray-lightest vads-u-padding--1p5">
+              <p className="vads-u-margin--0">Specialty care</p>
+              <p
+                id="facility-patient-satisfaction-scores-specialty-urgent-score"
+                className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
+              >
+                {this.state.facility.attributes.feedback.health
+                  .specialtyCareUrgent
+                  ? `${this.state.facility.attributes.feedback.health
+                      .specialtyCareUrgent * 100}%`
+                  : 'N/A'}
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <h3>Routine care appointments at this location</h3>
+        <p>
+          % of Veterans who say they usually or always get an appointment when
+          they need it
+        </p>
+        <div className="usa-grid-full">
+          <div className="usa-width-one-half">
+            <div className="usa-width-one-half vads-u-background-color--gray-lightest vads-u-padding--1p5">
+              <p className="vads-u-margin--0">Primary care</p>
+              <p
+                id="facility-patient-satisfaction-scores-primary-routine-score"
+                className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
+              >
+                {this.state.facility.attributes.feedback.health
+                  .primaryCareRoutine
+                  ? `${this.state.facility.attributes.feedback.health
+                      .primaryCareRoutine * 100}%`
+                  : 'N/A'}
+              </p>
+            </div>
+            <div className="usa-width-one-half vads-u-background-color--gray-lightest vads-u-padding--1p5">
+              <p className="vads-u-margin--0">Specialty care</p>
+              <p
+                id="facility-patient-satisfaction-scores-specialty-routine-score"
+                className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
+              >
+                {this.state.facility.attributes.feedback.health
+                  .specialtyCareRoutine
+                  ? `${this.state.facility.attributes.feedback.health
+                      .specialtyCareRoutine * 100}%`
+                  : 'N/A'}
```

Comment:
It might be nice to add a function for formatting as a percentage.

Reply:
do we have one already?

Reply:
Not that I know of! It could be a platform utility or just added to this component. Up to you 🙂

Reply:
alrighty, i'll add it as a utility :)